scalecube-organization/src/main/java/io/scalecube/config/ConfigRegistryConfiguration.java

```diff
@@ -38,10 +30,13 @@ public static ConfigRegistry configRegistry() {
 
     // for test purposes without vault access
     if (System.getenv().get("VAULT_ADDR") != null) {
-      builder.addLastSource("vault", VaultConfigSource.builder().build());
+      builder.addFirstSource("vault", VaultConfigSource.builder().build());

```

Comment:
it's confusing here, let's place it above of other declarations and leave it as `addLastSource`